spec/api-web-contents-spec.ts

```diff
@@ -1288,6 +1288,22 @@ describe('webContents module', () => {
     });
   });
 
+  describe('webrtc udp port range policy api', () => {
+    afterEach(closeAllWindows);
```

Comment:
In conjunction with my comments about silent failure above, we should also be testing invalid port ranges to ensure that they throw correctly and descriptively.